app/controllers/api/v1/identities_controller.rb

```diff
@@ -39,6 +39,14 @@ def update
         end
       end
 
+      def destroy
+        if identity.update_attribute(:status, 'deleted')
```

Comment:
Maybe we should have a model method #soft_delete! or something to do this, instead of calling #update_attribute.

Also in the model we could have a hash of statuses:

STATUSES = {
  deleted: 'deleted',
  active: 'active',
  ...
}

Then the #soft_delete! method could be:

def soft_delete!
  update_attribute(:status, STATUSES[:deleted])
end